docs/user/gcp/install_upi.md

```diff
@@ -24,6 +24,12 @@ Use a [staged install](../overview.md#multiple-invocations) to enable desired cu
 ### Create an install config
 Create an install configuration as for [the usual approach](install.md#create-configuration).
 
+If you are installing into a [Shared VPC (XPN)][sharedvpc],
+the installer will not be able to find the base domain zone in the host project.
+Create the `install-config.yaml` manually using the documentation references/examples.
+It must have `publish: Internal` to avoid the base domain verification when executing `create manifests` later in this workflow.
+This can be reversed in a step [below](enable-external-ingress-optional).
```

Comment:
this needs to be a separate section..

Reply:
67658b5fca0aeaa92b80e428086fdd30037dfc9d